PatientRegistrationApp/src/components/HouseholdContactInfo.vue

```diff
@@ -106,8 +108,13 @@ import customerSettings from '../customerSettings'
 			phoneTypeOptions:['Home', 'Mobile', 'Work'],
 			emailRules: [
 				(v) =>
-				/^[\s]*$|.+@.+\..+/.test(v) ||
-				"Please provide a valid e-mail address",
+				/^[\s]*$|.+@.+\..+/.test(v) || 'Please provide a valid e-mail address',
+			],
+			phoneNumberRules: [
```

Comment:
<minor> for internationalization purposes, might want to refactor this to be `phoneNumberRulesUS`. See: https://en.wikipedia.org/wiki/North_American_Numbering_Plan </minor>

Reply:
I changed the variable name to phoneNumberRulesUs